Minio.Functional.Tests/FunctionalTest.cs

```diff
@@ -1638,6 +1638,9 @@ private async static Task ListObjects_Test5(MinioClient minio)
                 Task[] tasks = new Task[numObjects];
                 for (int i = 1; i <= numObjects; i++) {
                     tasks[i - 1]= PutObject_Task(minio, bucketName, objectNamePrefix + i.ToString(), null, null, 0, null, rsg.GenerateStreamFromSeed(1));
+                    if (i % 100 == 0){
```

Comment:
Please add a comment here, so we will have the context when we look at the code later.